cmd/fleetctl/get.go

```diff
@@ -142,7 +142,7 @@ func printHostDetail(c *cli.Context, host *service.HostDetailResponse) error {
 	return printSpec(c, spec)
 }
 
-func printConfig(c *cli.Context, config *fleet.AppConfig) error {
+func printConfig(c *cli.Context, config *fleet.EnrichedAppConfig) error {
```

Comment:
I hope it's ok to print the AppConfig along with the extra information the the API returns? I.e. that this doesn't break a scenario where users run `fleetctl get config` and save that to a yaml file to use as actual config file later? I don't _think_ that would break this use case, as long as the extra sections are simply ignored when used (vs errors due to unknown sections).

Reply:
Keep in mind that (at least for now) it's just to run checks that fleet preview is configured as expected. If it turns out to raise too many concerns, I can simply remove those extra checks and just do those that use the AppConfig settings, leaving the output of `fleetctl get config` as it was before.

Reply:
`filters_by_team_and_paginates` is still flaky :( (hmm, unless I don't have the fix in this PR, not sure)

Reply:
Maybe we should add a flag `--include-server-config`, which could send a query param to the API, and THEN add those?

Reply:
Actually, we can decode with one or the other, depending on the flag, I just read we are already sending that in the API.

Reply:
Yeah, ok I'll go with that!